### PR TITLE
Add EQL track (#140)

### PR DESCRIPTION
**Pull request by matriv**
Add a new track to benchmark EQL performance.

Because of some sporadic errors with downloading the data the bucket url is
used is not the standard one, issue tracked in https://github.com/elastic/infra/issues/24746

(cherry picked from commit 44cfc28c02089976ad6cdac9c046830adf58611d)

### CODE REVIEW

File: eql/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## EQL track
+
+This track contains endgame data from SIEM demo cluster and can be downloaded from: https://rally-tracks.storage.googleapis.com/eql/endgame-4.28.2-000001-documents.json.bz2
+(The 1k test file can be downloaded from https://rally-tracks.storage.googleapis.com/eql/endgame-4.28.2-000001-documents-1k.json.bz2)
```
Comment (dliappis) [inline eql/README.md:4]: @matriv should we mention that this works only starting with ES `7.10.0`

**Review by dliappis — APPROVED** [review]
LGTM

@matriv i'd be great if you could add a comment about the minimum supported version in the README.

**Comment by matriv** [comment]
@dliappis is it enough?

**Review by dliappis — APPROVED** [review]
Thanks. LGTM.

**Pull request merged**